### PR TITLE
Fix crash caused UserAccountManager dependency not being injected

### LINKED ISSUE #4107: Nextcloud-dev crashed while open txt files
Hi nextcloudies,
since yesterday my nextcloud-dev app crashed if i try to open a txt file. Pics, odt, pdf etc works well. Also the 3.6.2 app from GStore works well. Could this caused by installation of carnet from nc app store yesterday?

### PR DESCRIPTION
**Pull request by ezaquarii**
Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

Resolves #4107

### CODE REVIEW

**Comment by ezaquarii** [comment]
@tobiasKaminsky @AndyScherzinger I think this should be merged ASAP.

File: src/main/java/com/owncloud/android/ui/preview/PreviewTextFragment.java [hunk 1]
```diff
@@ -67,7 +68,7 @@
 import androidx.appcompat.widget.SearchView;
 import androidx.core.view.MenuItemCompat;
 
-public class PreviewTextFragment extends FileFragment implements SearchView.OnQueryTextListener {
+public class PreviewTextFragment extends FileFragment implements SearchView.OnQueryTextListener, Injectable {
```
Comment (ezaquarii) [inline src/main/java/com/owncloud/android/ui/preview/PreviewTextFragment.java]: Missing `Injectable` caused the user account to be null and - surprise, surprise - NPE.
To test this code path just try opening any txt file. It should not crash.

**Comment by ezaquarii** [comment]
It looks like emulator died. There is no chance this fix causes a build failure.

**Comment by ezaquarii** [comment]
Build fails again, despite no errors in log.
Analysis is stuck for 90 minutes already...

**Comment by AndyScherzinger** [comment]
Latest build worked 👍

**Comment by ralfi** [comment]
Yeah, worked also here ...

**Comment by tobiasKaminsky** [comment]
Thank you @ezaquarii for finding the cause and also directly provide a PR :heart:

**Pull request merged**